pkg/server/token_minter.go

```diff
@@ -218,6 +199,12 @@ func (s *TokenMintServer) processRequest(r *http.Request, auditEvent *auditEvent
 	// Replace the requested repository list with actual values
 	request.Repositories = repos
 
+	logger.InfoContext(ctx, "generating token",
```

Comment:
@sethvargo will the logger print this correctly or should I turn this into JSON?

Reply:
tested, this works appropriately